docs/install/installing-istio.md

```diff
@@ -139,6 +139,101 @@ spec:
 EOF
 ```
 
+After you install the cluster local gateway, your service and deployment for the local gateway are both named `cluster-local-gateway`.
+You do not need to update the `config-istio` configmap under the `knative-serving` namespace, because Knative Serving can use
+the local gateway cluster-local-gateway by default.
+
+### Updating the `config-istio` configmap to use a non-default local gateway
+
+If you create a custom service and deployment for local gateway with a name other than `cluster-local-gateway`, you
+need to update gateway configmap `config-istio` under the `knative-serving` namespace.
+
+1. Edit the 1config-istio1 configmap:
+
+```shell
+kubectl edit configmap config-istio -n knative-serving
+```
+
+2. Replace the `local-gateway.knative-serving.cluster-local-gateway` field with the custom service. If you name both
```

Comment:
I'd add `As an example, ` in front of the `If you name both...` sentence so make it clear that `custom-local-gateway` is just an example.